### ISSUE TITLE
Invoke-MsBuild cannot find MSBuild when installing via Gallery

### ISSUE THREAD

**Issue by expor** [deadlydog/Invoke-MsBuild#51]
I don't know how to properly write this ticket but when installing Invoke-MsBuild 2.7.1 using `Install-Module -Name Invoke-MsBuild` I'm unable to build my project (with tracing turned on it looks like it doesn't even start searching for 64bit VS Code 2022, only x86). I checked the installed version and it's correct:

```
PS C:\Users\UserName > Get-InstalledModule

Version    Name                                Repository           Description
-------    ----                                ----------           -----------
2.7.1      Invoke-MsBuild                      PSGallery            Executes the MSBuild.exe tool against the specif...
2.3.0      7Zip4Powershell                     PSGallery            Powershell module for creating and extracting 7-...
```

However, when using `Import-Module -Name "C:\PathTo\Invoke-MsBuild.psm1` it works fine! I changed my script to use the local psm1 file now instead, but I don't understand why the normal Module doesn't work, as it should be the same thing?

**Comment by deadlydog** [comment]
Hmmm, that is very odd. The symptoms would make sense if it was defaulting to using an older version of Invoke-MsBuild, before it was updated to account for VS 2022 and 64-bit versions.

What is the output if you run:

> Get-Module Invoke-MsBuild -ListAvailable

Does it show multiple versions of Invoke-MsBuild installed? If so, try running:

> Uninstall-Module Invoke-MsBuild -RequiredVersion [Older.version.here]

**Comment by expor** [comment]
Indeed, I wasn't aware it would keep the old version when installing the new one. On top of that the initial version was installed when I used VS 2019, and my script only stopped working when I removed VS 2019 and moved to VS 2022. I saw there was a new `Invoke-MsBuild` so I just installed it, not knowing the old one also persists. Sorry, but thanks a lot for the help.
